src/Hosting/WindowsService/src/Microsoft.Extensions.Hosting.WindowsService.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>.NET Core hosting and startup infrastructures for Windows Services.</Description>
```

Comment:
This isn't exclusively for .NET Core, right? You could use it on .NET Framework too.